tests/console/libgcrypt.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that libgcrypt works as intended by executing
+#          the selftest and by calling libgcrypt-config.
+#          In addition to this test, also the gpg test should be
+#          checked by tester because gpg uses libgcrypt.
+# - execute libgcrypt self test
+# - using libgcrypt-config, check some parameters
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "opensusebasetest";
+use testapi;
+use utils;
+use strict;
+use warnings;
+use version_utils qw(is_sle is_opensuse);
+use registration;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run "rpm -q libgcrypt20";
+    if (script_run("rpm -q libgcrypt-devel") == 1) {
+        # sle-sdk can be added only for 12.3 and 12.4. On 15.x, the package is part of basesystem
+        if (is_sle("=12-SP3") || is_sle("=12-SP4") || is_sle("=12-SP5")) {
```

Comment:
This would be simpler with just `"<15"`

Reply:
ah right! During development I was dealing also with sp1 and sp2, but now I've schedule it only for sp3 and greater. I'll fix that